packages/datetime/src/dateRangeInput.tsx

```diff
@@ -43,12 +43,24 @@ import {
 export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {
 
     /**
+     * Whether the calendar popover should close when a date range is fully selected.
+     * @default true
+     */
+    closeOnSelection?: boolean;
+
+    /**
      * The default date range to be used in the component when uncontrolled.
      * This will be ignored if `value` is set.
      */
     defaultValue?: DateRange;
 
     /**
+     * Whether the component should be enabled or disabled.
```

Comment:
"Whether the text inputs are non-interactive." 

per other `disabled` props docs